src/main/java/ch/njol/skript/config/SectionNode.java

```diff
@@ -446,45 +446,74 @@ HashMap<String, String> toMap(final String prefix, final String separator) {
 		}
 		return r;
 	}
-	
+
 	/**
-	 * @param other
-	 * @param excluded keys and sections to exclude
-	 * @return <tt>false</tt> if this and the other SectionNode contain the exact same set of keys
+	 * Updates the values of this SectionNode based on the values of another SectionNode.
+	 * @param other The other SectionNode.
+	 * @param excluded Keys to exclude from this update.
+	 * @return True if there are differences in the keys of this SectionNode and the other SectionNode.
 	 */
 	public boolean setValues(final SectionNode other, final String... excluded) {
-		boolean r = false;
-		for (final Node n : this) {
-			if (CollectionUtils.containsIgnoreCase(excluded, n.key))
+		return setValues(other, false, excluded);
+	}
+
+	/**
+	 * Updates the values of this SectionNode based on the values of another SectionNode.
+	 * @param other The other SectionNode.
+	 * @param countValues Whether or not values should be counted.
+	 * If false, values from the other SectionNode will be copied to this one.
```

Comment:
I think this should be phrased differently. The main function of this method is to copy values over, so it should be said that if countValues is true, this is no longer the case. 

However, I'm also uncertain if the method `setValues` should have an argument to do exactly not what the name implies.